themes/classic/modules/contactform/views/templates/widget/contactform.tpl

```diff
@@ -112,6 +112,13 @@
       </section>
 
       <footer class="form-footer text-sm-right">
+        <style>
```

Comment:
oh man, why inline? :P

Reply:
As @alegout said, we wouldn't want heuristics to target inputs with a "hidden" class.
And we also want to make sure that other themes won't forget to hide that field :)